src/app/spaces/space.service.spec.ts

```diff
@@ -132,6 +132,57 @@ describe('Service: SpaceService', () => {
   let response = { data: responseData, links: {} };
   let expectedResponse = cloneDeep(responseData);
 
+  // for odd characters
+  let responseDataWithSlash: Space[] = [
+    {
+      name: 'Test/Space',
+      path: 'test/space',
+      teams: [],
+      defaultTeam: null,
+      'attributes': {
+        'name': 'Test/Space',
+        description: 'This is a space for unit test',
+        'created-at': null,
+        'updated-at': null,
+        'version': 0
+      },
+      'id': '1',
+      'type': 'spaces',
+      'links': {
+        'self': 'http://example.com/api/spaces/1'
+      },
+      'relationships': {
+        areas: {
+          links: {
+            related: 'http://example.com/api/spaces/1/areas'
+          }
+        },
+        iterations: {
+          links: {
+            related: 'http://example.com/api/spaces/1/iterations'
+          }
+        },
+        workitemtypegroups: {
+          links: {
+            related: 'http://example.com/api/spacetemplates/1/workitemtypegroups'
+          }
+        },
+        // collaborators: {
+        //   links: {
+        //     related: 'http://example.com/api/spaces/1/iterations'
+        //   }
+        // },
+        'owned-by': {
+          'data': {
+            'id': '00000000-0000-0000-0000-000000000000',
+            'type': 'identities'
+          }
+        }
+      }
+    }
+  ];
+  let responseWithSlash = { data: responseDataWithSlash, links: {} };
```

Comment:
Is this variable used anywhere, I can't find it.

Reply:
No; deleted.